lib/releaselib.sh

```diff
@@ -680,17 +680,13 @@ release::gcs::locally_stage_release_artifacts() {
 
   logecho "- Hashing files in ${gcs_stage##$build_output/}..."
   find "$gcs_stage" -type f | while read -r path; do
-    for bits in "256" "512"; do
-      sum="$(common::sha "$path" "$bits" "full")" || return 1
-      echo "$sum" > "${path}.sha${bits}"
-    done
-  done
+    common::md5 $path > "$path.md5" || return 1
+    common::sha $path 1 > "$path.sha1" || return 1
```

Comment:
I suggest we condition these two lines to run only if RELEASE_BRANCH is release-1.13, release-1.14, or release 1.15 and open an issue to remove this small block in six months when 1.15 goes out of support.